package.json

```diff
@@ -19,7 +19,7 @@
     "test": "yarn test:client"
   },
   "devDependencies": {
-    "babel-eslint": "^10.0.1",
+    "babel-eslint": "10.0.1",
```

Comment:
Why this change?

Reply:
Somehow babel-eslint@10.0.2 was installed. React script works with 10.0.1 only. That's why.